lib/hammer_cli_foreman_ansible/job_template.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module HammerCLIForemanAnsible
+  class JobTemplate < HammerCLIForeman::Command
+    resource :job_templates
+
+    HammerCLIForemanRemoteExecution::JobTemplate::InfoCommand.extend_output_definition do |definition|
+      definition.insert(:before, :description) do
+        field :ansible_callback_enabled, _('Ansible Callback Enabled'), Fields::Boolean
+      end
+    end
+  end
```

Comment:
While this works, it is probably not the cleanest possible way and it doesn't give us the power to solve 2) cleanly.

I'd suggest creating an extension class and then extending the info command with it as per[1]. It should also allow us to use `before_print` to conditionally hide the irrelevant fields

[1] - https://github.com/theforeman/hammer-cli/blob/master/doc/commands_extension.md

Reply:
If we do that, the extension class should probably live in `lib/hammer_cli_foreman_ansible/command_extensions/`

Reply:
I wasn't able to understand how the before_print works. I tried to debug it, but that also didn't work. 
This code works, but the `ansible_callback_enabled` is always displayed (for all types of providers).
Any idea how can I display it only for the ansible provider type?

Reply:
The before_print block receives three arguments, the last two are not useful to us here, but the first one is a hash of "raw" data that is about to be printed. If we delete the `ansible_callback_enabled` key from the hash, it will not get printed. The provider type is in there somewhere as well and we can decide whether to remove the callback field based on it.

Reply:
You need to do two things:
- mutate the object in place
- extend the original command class. Right now you're creating a subclass of it and extending that subclass, which doesn't really do much